test/contracts-network/colony-funding.js

```diff
@@ -95,10 +95,29 @@ contract("Colony Funding", (accounts) => {
       expect(pot2Balance).to.eq.BN(51);
     });
 
+    it("when moving tokens between pots, should respect permission inheritance", async () => {
```

Comment:
I wonder why this test didn't catch the `authDomain` issue.

Reply:
It didn't exist :stuck_out_tongue: